exercises/simple-linked-list/hints.md

```diff
@@ -0,0 +1,3 @@
+## Hints
+
+To support `list()`, see [implementing an iterator for a class.](https://docs.python.org/2/tutorial/classes.html#iterators)

```

Comment:
There's an implementation difference between Python 2 and Python 3 that I think needs to be made clear here to avoid confusion (`next()` vs `__next__()`).

I would suggest linking to the Python 3 documentation (I suspect that more people will be using Python 3) and then mentioning that `__next__` should be replaced by `next` if using Python 2, or that `next` can be implemented for dual compatibility as:

```python
def next(self):
    return self.__next__()
```

Reply:
That's exactly what I used in the example solution; I'll add the suggested info to `hints.md`